src/mobster/cli.py

```diff
@@ -130,20 +130,19 @@ def generate_product_parser(subparsers: Any) -> None:
     """
 
     product_parser = subparsers.add_parser(
-        "product", help="Generate an SBOM document for product"
+        "product", help="generate an SBOM document for product"
     )
-
     product_parser.add_argument(
-        "--data-path",
+        "--data",
         required=True,
         type=Path,
-        help="Path to the merged data file in JSON format.",
+        help="path to the merged data file in JSON format.",
```

Comment:
Suggestion: we should document the data format (the release notes json is expected, but just `data` is provided as a hint of usage).